### ISSUE TITLE
/sonarr_actions.py", line 194, in get_series_to_exclude KeyError: 'title'

### ISSUE THREAD

**Issue by stgreenb** [haijeploeg/excludarr#37]
**Describe the bug**
Bug when running in docker against large pool of sonarr shows. 

**To Reproduce**
See below

**Versions**
1. OS Version: ubutu
2. Excludarr version: current from dockerhub

**Debug logging**
Running 
```
docker run -it --rm --env-file excludarr.env haijeploeg/excludarr:latest --debug sonarr exclude -a not-monitored
```
gives me the following. Note it does process several shows (which I have omitted) before failing. 

```
[2021-12-27 16:12:41] - Starting Excludarr v1.0.2
[2021-12-27 16:12:41] - Got sonarr as subcommand
[2021-12-27 16:12:41] - Reading configuration file
[2021-12-27 16:12:41] - Determining which configfile to use
[2021-12-27 16:12:41] - Configfile to use: /etc/excludarr/excludarr.yml
[2021-12-27 16:12:41] - Reading configfile: /etc/excludarr/excludarr.yml
[2021-12-27 16:12:41] - Read the following configuration: {'general': {'fast_search': True, 'locale': 'en_
US', 'providers': ['netflix', 'amazon prime video', 'Disney Plus', 'Hulu']}, 'radarr': {'url': 'http://192.168.1.
197:7878', 'api_key': '<REDACTED>', 'verify_ssl': False, 'exclude': ['']}, 'sonarr': {'url': 'http://192.168.1.19
7:8989', 'api_key': '<REDACTED>', 'verify_ssl': False, 'exclude': ['']}}
[2021-12-27 16:12:41] - Got exclude as subcommand
[2021-12-27 16:12:41] - Got CLI values for -p, --provider option:
[2021-12-27 16:12:41] - Got CLI values for -l, --locale option: None
[2021-12-27 16:12:41] - Got CLI values for -a, --action option: not-monitored
[2021-12-27 16:12:41] - Got CLI values for -d, --delete option: False
[2021-12-27 16:12:41] - Got CLI values for -e, --exclusion option: False
[2021-12-27 16:12:41] - Got CLI values for -y, --yes option: False
[2021-12-27 16:12:41] - Got CLI values for --progress option: False
[2021-12-27 16:12:41] - Initializing PySonarr
[2021-12-27 16:12:41] - Initializing JustWatch API with locale: en_US
[2021-12-27 16:12:41] - Getting all the series from Sonarr
[2021-12-27 16:12:42] - Got the following providers: Netflix, Amazon Prime Video, Disney Plus, Hulu
```
```
Traceback (most recent call last):
  File "/usr/local/bin/excludarr", line 33, in <module>
    sys.exit(load_entry_point('excludarr==1.0.2', 'console_scripts', 'excludarr')())
  File "/usr/local/lib/python3.10/site-packages/excludarr-1.0.2-py3.10.egg/excludarr/main.py", line 68, in cli
  File "/usr/local/lib/python3.10/site-packages/typer/main.py", line 214, in __call__
    return get_command(self)(*args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/typer/main.py", line 500, in wrapper
    return callback(**use_params)  # type: ignore
  File "/usr/local/lib/python3.10/site-packages/excludarr-1.0.2-py3.10.egg/excludarr/commands/sonarr.py", line 69, in exclude
  File "/usr/local/lib/python3.10/site-packages/excludarr-1.0.2-py3.10.egg/excludarr/core/sonarr_actions.py", line 194, in get_series_to_exclude
KeyError: 'title'
```

**Comment by haijeploeg** [comment]
Duplicate of #30 has been fixed in #35 . Going to release v1.0.3 in a couple of minutes with the fix included.

**Comment by haijeploeg** [comment]
Version v1.0.3 is out!
